commands/http/websocketio.go

```diff
@@ -0,0 +1,57 @@
+package http
+
+import (
+	"io"
+	"time"
+
+	websocket "github.com/gorilla/websocket"
+)
+
+type WebsocketIO struct {
+	conn *websocket.Conn
+	reader io.Reader
+}
+
+func NewWebsocketIO(conn *websocket.Conn) WebsocketIO {
+	return WebsocketIO{conn, nil}
```

Comment:
i prefer:
```
return WebsocketIO{
    conn: conn,
}
```

probably also want to deal in pointers. (`*WebsocketIO` vs just `WebsocketIO`)